go/pkg/client/retries_test.go

```diff
@@ -219,7 +219,7 @@ func setup(t *testing.T) *flakyFixture {
 	f.client, err = client.NewClient(f.ctx, instance, client.DialParams{
 		Service:    f.listener.Addr().String(),
 		NoSecurity: true,
-	}, client.StartupCapabilities(false), client.ChunkMaxSize(2), client.RPCTimeouts(map[string]time.Duration{"default": 50 * time.Millisecond}))
+	}, client.StartupCapabilities(false), client.ChunkMaxSize(2), client.RPCTimeouts(map[string]time.Duration{"default": 50 * time.Millisecond, "Execute": 0}))
```

Comment:
Execute always times out?

Reply:
Execute should basically never have a timeout. This should have still worked, and it (and other operations) are the only bug that still remains (will fail in ~1/200 runs). Sometimes a call will time out of the 50ms before it reaches my fake server, and I don't know why.